### PR TITLE
libghthash 0.6.2

### PR DESCRIPTION
**Pull request by fd00**

### CODE REVIEW

File: Library/Formula/libghthash.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
+class Libghthash < Formula
+  homepage 'http://www.bth.se/people/ska/sim_home/libghthash.html'
```
Comment (mikemcquaid) [inline Library/Formula/libghthash.rb]: Use double-quotes everywhere (and generally run `brew audit --strict` for new formulae and fix all the warnings)

File: Library/Formula/libghthash.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          result = 1;
+        }
+        ght_finalize(p_table);
+
+        return result;
+      }
+    EOS
+    system ENV.cc, 'test.c', '-lghthash', '-o', 'test'
+    system 'ls', '-lh', 'test'
+    system 'file', 'test'
```
Comment (mikemcquaid) [inline Library/Formula/libghthash.rb]: Delete the `file` and `ls` tests, they aren't needed, thanks.

File: Library/Formula/libghthash.rb [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  def install
+    ENV.universal_binary if build.universal?
+    system 'autoreconf', '-ivf'
+    system './configure', '--disable-dependency-tracking',
+           "--prefix=#{prefix}"
+    system 'make', 'install'
+  end
+
+  test do
+    (testpath / 'test.c').write <<-EOS.undent
```
Comment (mikemcquaid) [inline Library/Formula/libghthash.rb]: `testpath/"test.c"`

**Comment by xu-cheng** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Pull request closed without merging**